### PR TITLE
Add basic html output to conan list

### PR DESCRIPTION
**Pull request by czoido**
Changelog: Feature: Add basic html output to `conan list` command.
Docs: omit

Adding the simplest html output to conan list command, just rendering the json as it is in the html
Also fixing a broken message when missing packages

Closes: https://github.com/conan-io/conan/issues/13022
Closes: https://github.com/conan-io/conan/issues/13021

### CODE REVIEW

File: conans/client/graph/install_graph.py [hunk 1]
```diff
@@ -285,10 +285,8 @@ def _raise_missing(missing):
...
-           Missing prebuilt package for '%s'
-           Use 'conan list packages %s --format=html -r=remote > table.html' and open the table.html file to see available packages
-           Or try to build locally from sources with '%s'
-
-           More Info at 'https://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package'
-           ''' % (missing_pkgs, ref, build_str)))
+        raise ConanException(textwrap.dedent(f'''\
+           Missing prebuilt package for '{missing_pkgs}'
+           Check the available packages using the 'conan list' command
+           or try to build locally from sources using the '{build_str}' argument
```
Comment (memsharded) [inline conans/client/graph/install_graph.py:291]: This change is what is most likely breaking a few tests.
- Are we sure we want to run without the link to the FAQ?
- Shouldn't we be explicit and add ``conan list pkg/version:* --format=html``?

Comment (SSE4) [reply conans/client/graph/install_graph.py:291]: why not run command here internally and print the output straight?

Comment (czoido) [reply conans/client/graph/install_graph.py:291]: I added: https://github.com/conan-io/docs/pull/2972 will link to it in the error message

File: conans/client/graph/install_graph.py [hunk 2]
```diff
@@ -285,10 +285,10 @@ def _raise_missing(missing):
...
-           Use 'conan list packages %s --format=html -r=remote > table.html' and open the table.html file to see available packages
-           Or try to build locally from sources with '%s'
+        raise ConanException(textwrap.dedent(f'''\
+           Missing prebuilt package for '{missing_pkgs}'
+           Check the available packages using 'conan list {ref}:* -r=remote'
+           or try to build locally from sources using the '{build_str}' argument
 
-           More Info at 'https://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package'
-           ''' % (missing_pkgs, ref, build_str)))
+           More Info at 'https://docs.conan.io/en/2/knowledge/faq.html#error-missing-prebuilt-package'
```
Comment (czoido) [inline conans/client/graph/install_graph.py:293]: Note that this url does not exist yet, this is were the docs will be published in the near future before we launch 2.0

**Review by memsharded — APPROVED** [review]

**Pull request merged**